test/puppetlabs/trapperkeeper/bootstrap_test.clj

```diff
@@ -46,21 +46,24 @@
               (is (= (hello-world hello-world-svc) "hello world")))))
 
         (testing "Gives precedence to bootstrap config in cwd"
```

Comment:
If the new approach is doing what I think it may be, then I wondered if it might be possible, and/or easy enough to test in a temp dir or at least in target/ so we don't have a test clobbering a file in the working tree.  In pdb we have some tests that test the uberjar itself from the outside (we run those separately in travis), which would make that eas(y|ier), but I'm guessing tk doesn't yet.  Might be too difficult for now, but would be nice to adjust if we can.

Reply:
Hey @rbrw , that's a great idea and when I saw this comment I started work on it. I've run into an issue that's currently blocking me (probably just needs a bit more debugging my system) but I'd like to set it down for now and revisit, so I can get this tk version into our 4.x clj-parent update and get info about it out to dev today. I've filed [TK-483](https://tickets.puppetlabs.com/browse/TK-483) which I'm hoping to do in a couple weeks when I investigate [SERVER-2421](https://tickets.puppetlabs.com/browse/SERVER-2421).